tests/MongoDB.Driver.Tests/Specifications/Runner/MongoClientJsonDrivenTestRunnerBase.cs

```diff
@@ -200,6 +205,11 @@ protected virtual void CustomDataValidation(BsonDocument shared, BsonDocument te
             // do nothing by default.
         }
 
+        protected virtual JsonDrivenTestFactory CreateJsonFactory(IMongoClient mongoClient, string databaseName, string collectionName, Dictionary<string, object> objectMap, EventCapturer eventCapturer)
```

Comment:
Not sure why this method is needed, but the name is wrong. Should be:

    CreateJsonDrivenTestFactory

or

    CreateTestFactory